src/ServicePulse.Host/vue/src/components/PaginationStrip.vue

```diff
@@ -115,7 +115,7 @@ const pages = computed(() => {
   <div v-if="showPagination" class="col align-self-center">
     <ul class="pagination justify-content-center">
       <li v-for="page of pages" class="page-item" :key="page.key">
-        <a class="page-link" @click="pageNumber = page.page" :class="page.class">{{ page.label }}</a>
+        <button :disabled="!!page.class?.disabled" :aria-label="page.label" class="page-link" @click="pageNumber = page.page" :class="page.class">{{ page.label }}</button>
```

Comment:
@mikeminutillo, following Testing Library [Guiding Principle](https://testing-library.com/docs/guiding-principles/), which encourages writing tests that closely resemble how web pages are used (even for component tests). I changed this template to render a button instead of an anchor so that the "toBeDisabled()" assertion can be used and to avoid checking for the usage of a CSS class, which is an implementation detail. Do you see any problem with this? @PhilBastian I checked how the strip renders, and looks ok, although there is a perceptible difference.

Reply:
That's fine. Phil and I discussed turning these into buttons. It is a systemic issue across the app. `<a>` tags are used without `href`. At the time we decided to leave it as is and do it across the board later.

Reply:
I have updated the styles to make this control look how it did in the old angular pages. Even before you changed it to be buttons it wasn't styling correctly